portfolio/src/main/webapp/index.html

```diff
@@ -116,13 +116,13 @@ <h2>Comments</h2>
        </div>
     </div>
 
-     <script src="https://www.gstatic.com/charts/loader.js"></script>
-        <script>
-        google.charts.load('current', {packages: ['corechart']});
-        google.charts.setOnLoadCallback(loadChart);
-        </script>
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script>
+    google.charts.load('current', {packages: ['corechart']});
+    google.charts.setOnLoadCallback(getPhrase);
+    </script>
 
-        <div id="activityPieChart"/>
+    <div id="commentBarChart"/>
```

Comment:
I think you should still open and close div tags rather than self close it

`<div id="commentBarChart"></div>`

Reply:
I'll change the close tags and push a commit